docs/platform/reference/eol-for-major-versions.rst

```diff
@@ -236,3 +239,58 @@ without committing their production services to a one-way upgrade.
    you see a ``New database fork`` button. This will allow you to make a
    separate new database service that is cloned from the current one's
    backups.
+
+Aiven tools
+-----------
+
+Alongside `Aiven Console <https://console.aiven.io/>`_, Aiven offers multiple tools for interacting with the Aiven platform and services. These include the Aiven CLI, the Aiven Terraform provider, and the Aiven Kubernetes® operator. Breaking changes in the Aiven API can result in new major versions of the Aiven tools. While backwards compatibility is typically maintained, certain changes require us to deprecate older versions of the tools.
+
+Aiven CLI
+'''''''''
+
+.. container:: intercom-interblocks-table-container
+
+   +-------------+---------------+
+   | **Version** | **Aiven EOL** |
+   |             |               |
+   +-------------+---------------+
+   | 1.x         | 2023-12-11    |
+   +-------------+---------------+
+   | 2.x         | 2023-12-11    |
+   +-------------+---------------+
+   | 3.x         | 2023-12-11    |
+   +-------------+---------------+
+   | 4.x         | `TBA*`        |
+   +-------------+---------------+
+
+Aiven Terraform provider
+''''''''''''''''''''''''
+
+.. container:: intercom-interblocks-table-container
+
+   +-------------+---------------+
+   | **Version** | **Aiven EOL** |
+   |             |               |
+   +-------------+---------------+
+   | 1.x         | 2023-12-31    |
+   +-------------+---------------+
+   | 2.x         | 2023-12-31    |
+   +-------------+---------------+
+   | 3.x         | 2023-12-31    |
+   +-------------+---------------+
+   | 4.x         | `TBA*`        |
+   +-------------+---------------+
+
+Aiven Kubernetes operator
+'''''''''''''''''''''''''
+
+.. container:: intercom-interblocks-table-container
+
+   +-------------+---------------+
+   | **Version** | **Aiven EOL** |
+   |             |               |
+   +-------------+---------------+
+   | 0.x         | `TBA*`        |
+   +-------------+---------------+
+   | 1.x         | `TBA*`        |
+   +-------------+---------------+
```

Comment:
@carabaestlein so are we adding the go client to the list?

Reply:
Not sure if we really need to do so, because we don't have much activity there. But to note, the current version is [v2](https://github.com/aiven/aiven-go-client/releases), and we don't develop v1 (backports, fixes, etc). Yet we accept PRs that happens like once or two a year.

Reply:
Let's leave this out until we have a clearer plan around how we want to more formally launch and then manage the go client!